site/_docs/reference.md

```diff
@@ -2768,7 +2771,7 @@ BigQuery's type system uses confusingly different names for types and functions:
 | m | TO_BASE64(string)                              | Converts the *string* to base-64 encoded form and returns a encoded string
 | b m | FROM_BASE64(string)                          | Returns the decoded result of a base-64 *string* as a string
 | b | TO_HEX(binary)                                 | Converts *binary* into a hexadecimal varchar
-| b | FROM_HEX(varchar)                              | Converts a hexadecimal-encoded *varchar* into bytes
+| b | FROM_HEX(varchar)                              | Converts a hexadecimal-encoded *varchar* into bytes, throws exception if input is invalid hexadecimal-encoded *varchar*
```

Comment:
Change `varchar` to `string`, and `,` to `;`, like this:

`; throws if *string* is not a valid hexadecimal string`

Reply:
Resolved.